store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/retriever/AccumuloSetRetriever.java

```diff
@@ -399,7 +408,7 @@ private boolean _hasNext() throws RetrieverException {
                 updateScanner();
             }
             if (!scannerIterator.hasNext()) {
-                scanner.close();
+                CloseableUtil.close(scanner);
```

Comment:
Worth noting that this swallows the exception whereas the previous method doesn't. Missed raising this in the original.

Reply:
Not sure... would you care about it not closing correctly? I'll do some digging...

Reply:
I am not sure either but it seems that, for now, this can just get merged and then be looked at as part of your closeable PR?

Reply:
Think maybe this should be looked at separately as suggested here: https://github.com/gchq/Gaffer/pull/2621#pullrequestreview-932162393